src/components/layout/AddToWishlistButton/AddToWishlistButton.js

```diff
@@ -4,49 +4,39 @@ import axiosInstance from '../../../apis/axiosClient'; // axios instance
 import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
 import { useNavigate } from 'react-router-dom';
 
-const wishlistEndpoint = '/wishlist';
-
-const AddToWishlistButton = ({ album }) => {
+const AddToWishlistButton = ({ album, wishListId }) => {
   const [isAdded, setIsAdded] = useState(false);
   const { status } = useAuth();
   const navigate = useNavigate();
   const isUserLoggedIn = status === AuthStatus.LoggedIn;
 
-  const fetchOrCreateWishlist = async () => {
-    try {
-      const response = await axiosInstance.post(wishlistEndpoint);
-      return response.data.wishlist._id; // Return the wishlist ID
-    } catch (error) {
-      console.error('Error creating wishlist:', error);
-    }
-  };
-
   useEffect(() => {
     const storedAddedStatus =
-      JSON.parse(localStorage.getItem(`addedStatus-${album._id}`)) || false;
+      JSON.parse(localStorage.getItem(`wishListAlbums-${album._id}`)) || false;
```

Comment:
If you do end up making the localstorage field a single object, then this will need to be updated to take that into account:

```suggestion
    const storedWishlist = JSON.parse(localStorage.getItem('wishlistAlbums')) || {};
    const storedAddedStatus = album._id in storedWishlist;
```

Reply:
Done